float64 Motor1
float64 Motor2
